src/state/reducers/quizReducers.js

```diff
@@ -71,24 +75,19 @@ export const quizReducer = (
       };
 
     case "QUIZ_SAVE_PENDING":
-      return { ...state, savePending: true, saveFailed: { message: "" } };
+      return { ...state, saveStatus: {type:"info", message: "Saving ..." } };
 
     case "QUIZ_SAVE_SUCEED":
       return {
         ...state,
-        quizCreation: false,
         data: { title: "", description: "", questions: [] },
-        savePending: false,
-        saveSuceed: true,
-        saveFailed: { message: "" },
+        saveStatus: {type:"info", message: "Quiz Saved !" },
       };
 
     case "QUIZ_SAVE_FAILED":
       return {
         ...state,
-        savePending: false,
-        saveSuceed: false,
-        saveFailed: { message: action.payload.message },
+        saveStatus: {type:"warning", message: action.payload.message },
       };
     case "GET_ALL_QUIZES":
       return { ...state, quizes: action.payload };
```

Comment:
the same is valid here... your repo would be better organized if this PR contained only styling-related code.
again... no need to change it now, just a note for future projects ;)